src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java

```diff
@@ -831,20 +830,26 @@ private static boolean isVariableInOperatorExpr(
      */
     private static boolean isChild(DetailAST parent, DetailAST ast) {
         boolean isChild = false;
-        final ASTEnumeration astList = parent.findAllPartial(ast);
```

Comment:
Just FYI, this is not a true conversion of the method. While looking for the AST, it also checks that the structure of AST (children) are the same. However all our tests use the ident for the AST so it will never have any children anyways.